### PR TITLE
[Migrator] Add fix-it tests to pick up swap exclusivity fix-its

### PR DESCRIPTION
**Pull request by bitjammer**
These are attached to warnings so they need to be added to the
opt-in list in the FixitFilter.

rdar://problem/31916085

### CODE REVIEW

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

File: test/Migrator/swap_fixit.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// RUN: rm -rf %t && mkdir -p %t && %target-swift-frontend -emit-migrated-file-path %t/swap_fixit.swift.result -enforce-exclusivity=checked -swift-version 3 -emit-sil -primary-file %s -o /dev/null -verify
```
Comment (devincoughlin) [inline test/Migrator/swap_fixit.swift]: I don't think you want the '-verify' here (which is why the bots are complaining).

**Review by devincoughlin — APPROVED** [review]
LGTM with the test change to remove '-verify'.

File: test/Migrator/swap_fixit.swift (hunk 1 above)
Comment (bitjammer) [inline test/Migrator/swap_fixit.swift]: Ah, that's what I get for copy/paste! Thanks!

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Comment by bitjammer** [comment]
Sorry for the testing noise, I'm currently trying to build other patches locally for something urgent.

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Comment by bitjammer** [comment]
Hm, weird. It didn't pick up the fix-it. Building locally now.

**Comment by bitjammer** [comment]
So, Devin tells me this is a SIL diagnostic, which is pretty obvious to me now considering I had to add the SILDiagnostics header. :-) It might be interesting to turn SIL generation on for the Migration pass but that significantly enlarges the scope of this patch.

**Pull request closed without merging**